### DISCUSSION TITLE
MPREMOTE over WIFI ?

### DISCUSSION THREAD

**Discussion by picpic020960** [micropython/micropython#10606] [General]
bonjour ,
Is it possible ?
Webrepl_cli.py is ok on wifi with thonny.
But mpremote is better : with wifi shall perfect !
( My tablette as no USB OTG ok for serial port so I'm facing a wall for dev)
Thanks for Reply
Regards

**Comment by rimelis** [comment]
Hello, 

You may consider to try _mpfshell_ (https://github.com/wendlers/mpfshell). It has support for WebREPL.

**Reply by picpic020960** [reply]
Thanks rimelis
But mpfshell has no 'mount' and no 'edit'
Definitedly mpremote wifi shall better
I Hope ...

**Comment by jimmo** [comment]
Adding webrepl support to mpremote is on the TODO list, after v1.20 is done.

**Reply by Wind-stormger** [reply]
Where can I see the TODO list ♿
